### DISCUSSION TITLE
https://zongzi531.com/2023/01/01/%EF%BC%92%EF%BC%90%EF%BC%92%EF%BC%93%E5%B9%B4%E3%81%8A%E3%82%81%E3%81%A7%E3%81%A8%E3%81%86%E3%81%94%E3%81%96%E3%81%84%E3%81%BE%E3%81%99/

### DISCUSSION THREAD

**Discussion by giscus** [zongzi531/zongzi531.github.io#43] [General]
# https://zongzi531.com/2023/01/01/%EF%BC%92%EF%BC%90%EF%BC%92%EF%BC%93%E5%B9%B4%E3%81%8A%E3%82%81%E3%81%A7%E3%81%A8%E3%81%86%E3%81%94%E3%81%96%E3%81%84%E3%81%BE%E3%81%99/

想先说的话疫情三年，国家保护我们免受新冠病毒的危害。
12月初，国家开放了，我也很快的感染上了新冠病毒。
...

https://zongzi531.com/2023/01/01/%EF%BC%92%EF%BC%90%EF%BC%92%EF%BC%93%E5%B9%B4%E3%81%8A%E3%82%81%E3%81%A7%E3%81%A8%E3%81%86%E3%81%94%E3%81%96%E3%81%84%E3%81%BE%E3%81%99/

**Comment by Shthah** [comment]
加油

**Reply by zongzi531** [reply]
感谢支持
